src/__swaps__/screens/Swap/components/SearchInput.tsx

```diff
@@ -38,9 +37,22 @@ export const SearchInput = ({
     return 'Close';
   });
 
-  const initialValue = useMemo(() => {
-    return SwapInputController.searchQuery.value;
-  }, [SwapInputController.searchQuery.value]);
+  const defaultValue = useMemo(() => {
+    return userAssetsStore.getState().searchQuery;
+  }, []);
+
+  const onSearchQueryChange = useCallback((event: NativeSyntheticEvent<TextInputChangeEventData>) => {
```

Comment:
also unrelated to the main focus of this pr, but i feel like this would be cleaner if it just accepts a string. that way you wouldn't be creating artificial NativeSyntheticEvents below